ush/config_defaults.yaml

```
@@ -2962,6 +2976,37 @@ task_aqm_lbcs:
       tasks_per_node: 24
       walltime: 00:30:00
 
+# ------------------------------------
+# SRW-AQM MELODIES-MONET Configuration
+# ------------------------------------
+task_mm_prep:
+  execution:
+    batchargs:
+      nodes: 1
+      tasks_per_node: 1
+      walltime: 00:05:00
+  # Output directory for MELODIES MONET-generated model evaluation plots and statistics.
+  MM_OUTPUT_DIR: null
+  # Evaluation packages to initialize and run.
+  MM_EVAL_PACKAGES: [chem]
+  # If set to another SRW experiment path, MELODIES MONET will generate "scorecards" for model inter-
+  # comparison.
+  MM_BASE_MODEL_EXPT_DIR: null
+  # Path, optionally with wildcards, selecting the AirNow observation files used by MELODIES MONET.
+  # For example: /staged/obs/Observations/AirNow/AirNow_2023*.nc
+  MM_OBS_AIRNOW_FN_TEMPLATE: null
+task_mm_run:
+  execution:
+    batchargs:
+      nodes: 1
+      tasks_per_node: !int '{{ platform.NCORES_PER_NODE }}'
+      # More time is needed if there is an additional model is included.
+      walltime: '{% if task_mm_prep.MM_BASE_MODEL_EXPT_DIR is none %}00:20:00{% else %}00:40:00{% endif %}'
```

It looks like 00:20:00 is too little time to complete the `mm_run_chem_stats` task on Orion (and possibly Hera, since the nodes are older than the rest of the RDHCPS platforms).  Increasing this time by ten minutes allowed the task to successfully complete (on Orion):
```
202306021200       mm_run_chem_stats                    21061272           SUCCEEDED                   0         1        1688.0
```
```suggestion
      walltime: '{% if task_mm_prep.MM_BASE_MODEL_EXPT_DIR is none %}00:30:00{% else %}01:00:00{% endif %}'
```